src/library/export/trackexportdlg.cpp

```diff
@@ -63,6 +97,7 @@ void TrackExportDlg::slotProgress(const QString& filename, int progress, int cou
     exportProgress->setValue(progress);
 }
 
+//TODO review this code to see how I can make a dialog box ask the user whether they want to skip importing the file or not
```

Comment:
Comments are not intended for personal notes. Please leave the original comment in place until you change the code.